roles/iptables/tasks/iptables.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: iptables variables
+  include_vars: vars/iptables.yml
+  tags: iptables-vars
+  when: open_tcp_ports is undefined
+
+- name: iptables chain setup
+  become: yes
+  script: ../scripts/iptables_setup.sh
+
+- name: iptables template setup
+  become: yes
+  template:
+    src: templates/iptables.j2
```

Comment:
Also, can you put the template at the same filesystem location as the destination? For example, take a look at the archive role's template task.